### ISSUE TITLE
Add feature: contrast toggle

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#21]
Player should be able to toggle contrast in the sidebar

**Comment by moates** [comment]
I'm nixing this feature. 

It would require me to create a new stylesheet for each color combo we have. Using a simple [invert() filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) is expensive and buggy in the few tests I tried.  

Color inversion is frequently available on a user's device or at the browser level anyway. People that need that feature are likely to be already aware of it.
